src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -172,6 +185,15 @@ public GeoApiContext setEnterpriseCredentials(String clientId, String cryptograp
   }
 
   /**
+   * Sets the default channel for requests (can be overridden by requests).  Only useful for Google Maps for Work clients;
```

Comment:
I believe this should be 'Only useful for Google Maps for Work clients.'

Thanks!